data/models/change.py

```diff
@@ -0,0 +1,98 @@
+"""
+💡 QUOI:
+Modèles pydantic liés aux changements atomiques
+apportés par les suggestions
+
+🎯 POURQUOI:
+ - pydantic nous permet de découpler les suggestions
+   de la DB
+ - mais la méthode .apply() nous permet d'utiliser
+    les modèles qfdmo pour appliquer les changements
+    au moment voulu
+
+🚧 COMMENT APPLIQUER UN CHANGEMENT:
+ - Faire passer la donner de suggestion dans le modèle
+ - Appeler .apply()
+
+
+🧊 MODELISATION DES SUGGESTIONS:
+On pourrait stocker les changements des suggestions
+sous forme modélisée:
+ 🟢 avantage: prêt a l'emploi (plus qu'a récupérer et faire apply
+ au moment de l'approbation), mais au final gain marginal
+ 🔴 incovénient: les suggestions deviennent dépendentes
+ des modèles, si les modèles changent on perd potentiellement
+ les suggestions OU il faut en faire des migrations
+
+Pour l'instant on voit que 🔴>>🟢 et on décide de:
+ 1) suggestion: on utilise le modèle UNIQUEMENT pour la validation,
+    mais on stoque la donnée non modèlisée
+ 2) changement: on rejoue la donnée dans le modèle et on applique .apply()
+"""
+
+from pydantic import BaseModel
+
+from qfdmo.models import RevisionActeur
+
+
+class ChangeActeurUpdateParent(BaseModel):
+    """Un acteur est rattaché à un parent,
+    l'acteur peut avoir ou pas un parent existant"""
+
+    identifiant_unique: str
+    parent_id: str
+
+    def apply(self):
+        _ = RevisionActeur.objects.filter(identifiant_unique=self.identifiant_unique)
+        # etc...
```

Comment:
je suppose que cette fonction n'est pas implémentée, dans ce cas pour éviter la confusion, je te propose d'écrire

soit 

```suggestion
        # TODO : this function is not implemented yet
        pass
```

soit 

```suggestion
        raise Exception("function not implemented yet")
```

Reply:
J'ai tout mis en `NotImplementedError` car c'était du pseudocode pour montrer l'idée